### PR TITLE
Fixed issue with leading zeros in Viz Post Processing

### PR DESCRIPTION
**Pull request by nickchadwick-noaa**
When extracting the feature_ids from the `hydroTable.csv` and adding them to the `src.json`, as per Viz's output requirements, there was an issue with HydroIDs having leading zeros in the `hydroTable.csv`, but not in the `src.json`.

This was fixed by stripping the leading zeros from the HydroIDs in the `hydroTable.csv` when comparing them to the `src.json`.

With this fixed, when using the `-v` parameter, the `rating_curves_<HUD_ID>.json` should now correctly have the `nwm_feature_id` property added to each HydroID.

This fix only affects the `-v` post-processing parameter and does not affect any metrics regarding the fim output.

Some good test hucs that have HydroIDs with leading zeros are:

 - 030201
 - 060101
 - 080402

### CODE REVIEW

**Comment by BradfordBates-NOAA** [comment]
This appears to solve the bug, although I recommend providing Viz (@LauraKeys-NOAA) with sample data to test before merging this branch.

**Comment by LauraKeys-NOAA** [comment]
Looks good to me.

**Comment by BradfordBates-NOAA** [comment]
Thanks Laura. I'll approve after you confirm that the Viz scripts ingest these data with no issues.

**Comment by LauraKeys-NOAA** [comment]
It looks like we're good on the viz side reading in all the files and using
them.

**Review by BradfordBates-NOAA — APPROVED** [review]

**Pull request merged**
